libraries/SRV_Channel/SRV_Channel_aux.cpp

```
@@ -72,19 +72,23 @@ void SRV_Channel::output_ch(void)
  */
 void SRV_Channels::output_ch_all(void)
 {
+#if NUM_SERVO_CHANNELS
```

from dev call, preference is for a FOR_EACH_SERVO_CHANNEL macro

---

peter is happy with pragam for suppressing error instead